.ci/travis/travis.sh

```diff
@@ -710,6 +710,16 @@ checkstyle-cli-run-openjdk14)
   exit $RESULT
   ;;
 
+spotbugs-and-pmd)
+  export MAVEN_OPTS='-Xmx2000m'
+  mvn -e clean test-compile pmd:check spotbugs:check
+  grep "Processing_Errors" target/site/pmd.html | cat > errors.log
+  echo "Errors are detected in target/site/pmd.html."
+  RESULT=$(cat errors.log | wc -l)
+  cat errors.log
+  if [[ $RESULT != 0 ]]; then false; fi
```

Comment:
do `sleep 5s` before `false`.
to let Travis flush message to log before container is failed and destroyed


please print `echo "Errors are detected in target/site/pmd.html."` in IF.

Reply:
Fixed.